node/pkg/logscribeconsumer/types.go

```diff
@@ -0,0 +1,73 @@
+package logscribeconsumer
+
+import (
+	"time"
+
+	"bisonai.com/orakl/node/pkg/logscribe/api"
+	"github.com/rs/zerolog"
+)
+
+const (
+	DefaultLogStoreInterval     = 10 * time.Second
+	DefaultBufferSize           = 3000
+	DefaultMinimalLogStoreLevel = zerolog.WarnLevel
+	DefaultLogscribeEndpoint    = "http://orakl-logscribe.orakl.svc.cluster.local:3000/api/v1/"
+	ProcessLogsBatchSize        = 1000
+)
+
+type App struct {
+	StoreInterval     time.Duration
+	buffer            chan map[string]any
+	consoleWriter     zerolog.ConsoleWriter
+	LogscribeEndpoint string
+	Service           string
+	Level             int
```

Comment:
how about using zerolog.Level type? can you check if it's valid?
```suggestion
	Level             zerolog.Level
```

Reply:
I think using int on app instance should be okay because it's not exposed to the caller/user. The `Level` field in the`AppConfig` struct is `String` so that the caller can pass `debug`, `info`, etc. The `Level` field in the `App` struct is only used for internal processing and posting to `logscribe`. 

Or do you mean we should update the `level` type in db to `String`?

Reply:
I was thinking that typecasting in app lint 150 was unnecessary
```
return int(lv) >= a.Level
```
it would be better if it can be compared directly
```
return lv >= a.Level
```

Reply:
i see, good point, will make an update